packages/ui/components/authors/team.ts

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright 2023 Janus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export const JANUS_IDP_TEAM = {
+  sabre1041: {
+    name: 'Andrew Block',
+    twitterUsername: 'sabre1041',
+    picture: '/images/people/sabre1041.jpg',
```

Comment:
Do we want to keep a copy of the pictures or use the GitHub profile picture url?

Reply:
We should try to keep all static content locally on the server instead of fetching it from elsewhere.